.github/workflows/docker.yml

```
@@ -0,0 +1,60 @@
+name: Docker
+
+on:
+  push:
+    tags: ['*']
+
+jobs:
+  build:
+    name: Build and push Waltz image
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: waltz
+          POSTGRES_PASSWORD: postgres 
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s 
+          --health-timeout 5s 
+          --health-retries 10
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'adopt'
+
+    - name: Cache Maven packages    
+      uses: actions/cache@v1
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-m2
+        restore-keys: ${{ runner.os }}-m2
+
+    - name: Login to GitHub Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Build with Maven - postgresql
+      run: mvn -B -s .build.settings.xml -Pci,build-postgres,waltz-postgres clean package;
+
+    - name: Build Docker image
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        push: true
+        tags: |
+          ghcr.io/${{ github.repository }}:${{ github.ref_name }}
+          ghcr.io/${{ github.repository }}:postgresql-${{ github.ref_name }}
+          ghcr.io/${{ github.repository }}:latest
```

there should also be `postgresql` as tag.
```suggestion
          ghcr.io/${{ github.repository }}:latest
          ghcr.io/${{ github.repository }}:postgresql
```